docs/api/wallet-rpc.yaml

```diff
@@ -1120,6 +1120,12 @@ components:
           type: integer
           example: 6
           description: Bitcoin miner fee to use for transaction. A number higher than 1000 is used as satoshi per kvB tx fee. The number lower than that uses the dynamic fee estimation of blockchain provider as confirmation target.
+        selected_utxos:
+          type: array
+          items:
+            type: string
+            example: 85cf4c880876eead0a6674cbc341b21b86058530c2eacf18a16007f8f9cb1b1a:0
+          nullable: true
```

Comment:
This may be the line that breaks compatibility. Parameter is simply not required, and beside compatibility issue it makes no sense to supply the parameter with a null value.

Reply:
Hey @roshii, I appreciate your feedback, and thanks for catching that! I've updated the PR and removed `nullable: true` since it wasn't serving any purpose. However, I don't believe that's the reason behind the `OpenAPI Diff` failure.

Reply:
That wasn't the reason for the failure indeed. OpenAPI Diff still catches `selected_utxos` as required 🤔 
I will dig into it soon as as I can.